nms/app/views/alarms/components/alertmanager/Receivers/Receivers.tsx

```diff
@@ -59,6 +65,52 @@ const receiverColumns = [
   },
 ];
 
+function ReceiverFilter(props: {
+  onAddReceiverClick: () => void;
+  showGlobalConfig: boolean;
+  setShowGlobalSettings: () => void;
+}) {
+  return (
+    <Grid container justifyContent="flex-end" alignItems="center" spacing={2}>
+      {props.showGlobalConfig === true && (
+        <Grid item>
+          <Button
+            onClick={() => props.setShowGlobalSettings()}
+            startIcon={<SettingsIcon />}
+            variant="outlined">
+            Settings
+          </Button>
+        </Grid>
+      )}
+      <Grid item>
+        <Button
+          data-testid="add-receiver-button"
+          variant="contained"
+          color="primary"
+          onClick={() => props.onAddReceiverClick()}>
+          Create Receiver
+        </Button>
+      </Grid>
+    </Grid>
+  );
+}
+
+function ReceiverEmpty(props: {onAddReceiverClick: () => void}) {
+  const classes = useStyles();
+  return (
+    <Grid container direction="column" alignItems="center">
+      <div className={classes.emptyReceiverTitle}>No Receiver Added</div>
+      <Button
+        size="small"
+        variant="contained"
+        color="primary"
+        onClick={() => props.onAddReceiverClick()}>
+        Create Receiver
+      </Button>
```

Comment:
If we have a grid here, should the button not be enclosed as `grid item`?
With the abstraction suggested above, this would probably come "for free".